Python/ceval.c

```diff
@@ -4914,7 +4945,7 @@ MISS_WITH_CACHE(LOAD_GLOBAL)
 MISS_WITH_CACHE(LOAD_METHOD)
 MISS_WITH_CACHE(CALL_FUNCTION)
 MISS_WITH_CACHE(BINARY_OP)
-MISS_WITH_OPARG_COUNTER(BINARY_SUBSCR)
+MISS_WITH_CACHE(BINARY_SUBSCR)
```

Comment:
I'm guessing it's still worth keeping the logic for oparg counters, just in case we implement a new family that can uses it in the future?

Reply:
Might as well delete it. It's in the git history.